server/static/tree/tree.html

```diff
@@ -36,7 +36,8 @@
 				</div>
 				<div id="error"></div>
 			</div>
-
+			<p>Logged in as: <span id="username"></span></p>
```

Comment:
Can we re-style this so it's just the logoutButton, but says "logout <username>", and have the button aligned to the bottom of the tree map box, on its right under the filters?

This reduces its importance a bit (most users will never click this button), and reduces the risk of an accidental click when interacting with the filters.